debian.Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:18.04
```

Comment:
Why do you need this dockerfile ?

Reply:
This is literally what builds the package...